src/openfl/display/SubBitmapData.hx

```diff
@@ -73,8 +74,6 @@ class SubBitmapData extends BitmapData {
 		readable = false;
 		image = null;
 
-		var gl = GL.context;
```

Comment:
don't want to access statics that shouldn't exist in the first place :) so instead we forward the `getTextrue` call to the parent and not store texture/textureContext at all